src/app/models/theme.ts

```diff
@@ -0,0 +1,5 @@
+export enum Theme {
```

Comment:
Really like that this was done in such a way that additional themes could theoretically be added.